tools/run_container.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# This tool attempts to detect the presence of various tools to run the CI
+# images. If present, it'll use them to run the specified container image.
+
+function rc_buildah() {
+    buildah_path="$(command -v buildah)"
+    podman_path="$(command -v podman)"
+
+    [ "x$buildah_path" != "x" ] && [ "x$podman_path" != "x" ]
+}
+
+function rc_docker() {
+    docker_path="$(command -v docker)"
+
+    [ "x$docker_path" != "x" ]
+}
+
+function rc_run() {
+    local image="$1"
+    local ret=0
+
+    if [ ! -f "tools/Dockerfiles/$image" ]; then
+        echo "Error: tools/Dockerfiles/$image is not a file; must be an" 1>&2
+        echo "existing location to launch container." 1>&2
+
+        exit 1
+    fi
+
+    if rc_buildah; then
```

Comment:
Do we support both docker and buildah? I see only Dockerfiles along with this PR. Is buildah better than Docker?